README.md

```diff
@@ -1,48 +1,70 @@
-provide any number of additional DataLad commands that are automatically
-included in DataLad's command line and Python API.
-
-For a demo, clone this repository and install the demo extension via
-
-    pip install -e .
-
-DataLad will now expose a new command suite with a `hello...` command.
-
-    % datalad --help |grep -B2 -A2 hello
-    *Demo DataLad command suite*
-
-      hello-cmd
-          Short description of the command
-
-To start implementing your own extension, [use this
-template](https://github.com/datalad/datalad-extension-template/generate), and
-adjust as necessary. A good approach is to
-
-- Pick a name for the new extension.
-- Look through the sources and replace `datalad_fuse` with
-  `datalad_<newname>` (hint: `git grep datalad_fuse` should find all
-  spots).
-- Delete the example command implementation in `datalad_fuse/__init__.py`
-  by (re)moving the `HelloWorld` class.
-- Implement a new command, and adjust the `command_suite` in
-  `datalad_fuse/__init__.py` to point to it.
-- Replace `hello_cmd` with the name of the new command in
-  `datalad_fuse/tests/test_register.py` to automatically test whether the
-  new extension installs correctly.
-- Adjust the documentation in `docs/source/index.rst`.
-- Replace this README.
-- Update `setup.cfg` with appropriate metadata on the new extension.
-
-You can consider filling in the provided [.zenodo.json](.zenodo.json) file with
-contributor information and [meta data](https://developers.zenodo.org/#representation)
-to acknowledge contributors and describe the publication record that is created when
-[you make your code citeable](https://guides.github.com/activities/citable-code/)
-by archiving it using [zenodo.org](https://zenodo.org/). You may also want to
-consider acknowledging contributors with the
-[allcontributors bot](https://allcontributors.org/docs/en/bot/overview).
+# DataLad FUSE extension package
+
+[![Build status](https://ci.appveyor.com/api/projects/status/g9von5wtpoidcecy/branch/master?svg=true)](https://ci.appveyor.com/project/mih/datalad-extension-template/branch/master) [![codecov.io](https://codecov.io/github/datalad/datalad-fuse/coverage.svg?branch=master)](https://codecov.io/github/datalad/datalad-fuse?branch=master) [![tests](https://github.com/datalad/datalad-fuse/workflows/Test/badge.svg)](https://github.com/datalad/datalad-fuse/actions?query=workflow%3ATest) [![docs](https://github.com/datalad/datalad-fuse/workflows/docs/badge.svg)](https://github.com/datalad/datalad-fuse/actions?query=workflow%3Adocs)
+
+`datalad_fuse` provides commands for reading files in a
+[DataLad](http://datalad.org) dataset from their remote web URLs without having
+to download them in their entirety first.
+
+## Installation
+
+`datalad_fuse` requires Python 3.6 or higher.  Just use
+[pip](https://pip.pypa.io) for Python 3 (You have pip, right?) to install it:
+
+    python3 -m pip install datalad_fuse
```

Comment:
although I like consistency between package name on pypi and python package/module, we usually use `-` for such names in datalad (and other) world. Let's call it `datalad-fuse` on pypi for consistency.  And `pip` seems to not care in `-` vs `_` since `pip install datalad_container` also works, right?

Reply:
Done.